site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -20,7 +35,9 @@ contributes towards a common purpose.
 User interfaces should be minimal and have intent. They can range from a simple icon, such as the [Google
 Mail Checker extension][2] shown on the right, to [overriding][3] an entire page.
 
-Extension files are zipped into a single `.crx` package that the user downloads and installs. This
+### How do users get extensions?
+
+When you make an extension, you zip it into a single `.crx` package that the user downloads and installs. This
 means extensions do not depend on content from the web, unlike ordinary web apps.
```

Comment:
I suspect that the original version used the passive voice because CWS is typically the entity that generates the crx package that a user downloads.

I think it may be better to pivot this section (+1 to creating a new heading) a little to focus more on CWS. While we may want to talk about what a CRX file is, I'm not sure it fits under this heading. To that end here's a quick attempt to draft up a new body for this section.

> Most Chrome users get extensions from the Chrome Web Store (CWS). Developers across the globe submit their extension to the Chrome Web Store where they are reviewed and made available to end users. 
> 
> Some businesses and schools use [enterprise policies](https://cloud.google.com/docs/chrome-enterprise/policies) to install extensions on their user's devices. These extensions are either fetched from the Chrome Web Store or they may be hosted on web servers. See XXX for info on the capabilities and limitations of  self hosting.

Reply:
Adapted the suggested text to replace the content of this section.